cmake/packaging/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+project("Devkit-Test")
+
+cmake_minimum_required(VERSION 3.18)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+
+set(PLAIDML_INSTALL_DIR /path/to/PlaidML)
+
+# Include PlaidML package
+include("${PLAIDML_INSTALL_DIR}/share/cmake/PlaidMLConfig.cmake")
+
+# Include 3rd party packages necessary for edsl_tests
+include(FetchContent)
+include("${PLAIDML_INSTALL_DIR}/share/cmake/third_party/gflags.cmake")
+include("${PLAIDML_INSTALL_DIR}/share/cmake/third_party/googletest.cmake")
+include("${PLAIDML_INSTALL_DIR}/share/cmake/third_party/half.cmake")
```

Comment:
I kinda think we should just copy/paste these or perhaps do a compile-time include here so that devkit users don't have to deal with these extra `.cmake` files